src/kv/store.h

```diff
@@ -749,6 +749,14 @@ namespace kv
             return success;
           }
           auto h = get_history();
+          if (!h->verify(term_))
```

Comment:
The paths for CFT and BFT seem pretty much identical now. Can we refactor?

Reply:
Unfortunately they are not :( This is the smallest/simplest variant of the code I have come up with.